code/game/mecha/mecha.dm

```diff
@@ -718,7 +718,7 @@
 
 /obj/mecha/emp_act(severity)
 	if(get_charge())
-		use_power((cell.charge/2)/severity)
+		cell.emp_act((severity*0.75))
```

Comment:
```suggestion
		cell.emp_act(round(severity*0.75))
```
Non-integer severity is not implemented in most cases.

Reply:
but then this doesn't work?

Reply:
...Actually, yeah, you're right, round() just returns severity to its original value for pretty much every EMP source in the game.

Reply:
Since this doesn't end up calling emp_act on anything except power cells, which correctly handle floating point severity, I guess it's "fine".
However, lower `severity` counter-intuitively has a greater effect, with 2 being the weaker option and 1 being the stronger, so you probably want to increase the value rather then decrease it here.
I think you need to rethink this, and then test it.

Reply:
Yeah, I only recently noticed that. I'm fine-tuning it, should be good to go later tonight.